UKHomeOffice/cerberus-service
# COP-10750: Add test for Clean up orphaned rules if all task links are removed

## Description

Add test for Clean up orphaned rules if all task links are removed
## To Test
npm run cypress:test:local -- --spec cypress/integration/airpax/delete-tasks.spec.js

Should delete an airpax task with movementId and verify the RuleId is also deleted

## Developer Checklist

\* Required
